blas/gonum/level2double.go

```diff
@@ -58,14 +58,10 @@ func (Implementation) Dgemv(tA blas.Transpose, m, n int, alpha float64, a []floa
 	}
 
 	var kx, ky int
-	if incX > 0 {
-		kx = 0
-	} else {
+	if incX <= 0 {
```

Comment:
`incX == 0` also results in `kx = 0`.  Makes more sense to use `<` test here.

Same in the other changes.

Reply:
Good point. I was doing a blind inversion.